# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../gpdf/hotfix.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./xpdf/TextOutputDev.h.orig	2007-04-18 12:53:48.000000000 +0000
+++ ./xpdf/TextOutputDev.h	2007-04-18 12:54:24.000000000 +0000
@@ -26,6 +26,10 @@
 class GfxState;
 class UnicodeMap;
 
+class TextBlock;
+class TextPage;
+class TextLineFrag;
+
 //------------------------------------------------------------------------
 
 typedef void (*TextOutputFunc)(void *stream, char *text, int len);
